src/pybind/nnet3/nnet_nnet_pybind.cc

```diff
@@ -19,31 +19,90 @@
 
 #include "nnet3/nnet_nnet_pybind.h"
 
-#include "nnet3/nnet-nnet.h"
 #include "nnet3/nnet-component-itf.h"
+#include "nnet3/nnet-convolutional-component.h"
+#include "nnet3/nnet-nnet.h"
+#include "nnet3/nnet-normalize-component.h"
+#include "nnet3/nnet-simple-component.h"
 
 using namespace kaldi;
 using namespace kaldi::nnet3;
 
+template <typename... Args>
+using overload_cast_ = py::detail::overload_cast_impl<Args...>;
+
 void pybind_nnet_nnet(py::module& m) {
-     using PyClass = Nnet;
-     py::class_<PyClass>(
-          m, "Nnet",
-          "This function can be used either to initialize a new Nnet from a config"
-          "file, or to add to an existing Nnet, possibly replacing certain parts of"
-          "it.  It will die with error if something went wrong."
-          "Also see the function ReadEditConfig() in nnet-utils.h (it's made a"
-          "non-member because it doesn't need special access)."
-     )
-     .def(py::init<>())
-     .def("Read", &PyClass::Read, py::arg("is"), py::arg("binary"))
-     .def("GetComponentNames", &PyClass::GetComponentNames, 
-     "returns vector of component names (needed by some parsing code, for instance).",
-     py::return_value_policy::reference)
-     .def("GetComponentName", &PyClass::GetComponentName, py::arg("component_index"))
-     .def("Info", &PyClass::Info)
-     .def("NumComponents", &PyClass::NumComponents)
-     .def("NumNodes", &PyClass::NumNodes)
-     // .def("GetComponent", &PyClass::GetComponent, py::arg("c"))
-     ;
+  using Comp = kaldi::nnet3::Component;
+  py::class_<Comp>(m, "Component",
```

Comment:
`Component` is from the file `nnet-component-itf`. Please create a new file
to wrap it.

When a class in Kaldi is in `foo.h`,  please wrap it in `foo_pybind.h`
and `foo_pybind.cc`